### PR TITLE
Minor cleanup around widget catalog

### PR DESCRIPTION
**Pull request by Hixie**
cc @sethladd

### CODE REVIEW

**Comment by sethladd** [comment]
@FaisalAbid any clues as to why the auto-stager didn't work here?

File: _includes/catalogpage.html [hunk 1]
```diff
@@ -22,9 +22,7 @@
         {% if comp.categories contains include.category %}
         <li class="cards__item">
             <div class="catalog-entry">
-                <div class="catalog-image-holder">
-                    {% if comp.image contains '<svg' %}{{comp.image}}{% else %}<img alt="" src="{{comp.image}}" />{% endif %}
-                </div>
+                <div class="catalog-image-holder">{{comp.image}}</div>
```
Comment (sethladd) [inline _includes/catalogpage.html]: we don't need the `` tag here?

Comment (FaisalAbid) [inline _includes/catalogpage.html]: @Hixie Images can be SVG or png's etc, so this would render just the url in many cases

**Comment by FaisalAbid** [comment]
@sethladd Not sure, the code didnt even execute looking at travis, will take a look

**Comment by FaisalAbid** [comment]
@sethladd it looks like a change in travis

```Travis CI makes encrypted variables and data available only to pull requests coming from the same repository. These are considered trustworthy, as only members with write access to the repository can send them.

Pull requests sent from forked repositories do not have access to encrypted variables or data.```

Since @Hixie sent it from a forked repo, all the encrypted variables in Travis are removed, including the ones PR bot uses.

**Comment by sethladd** [comment]
Ah! Good to know. Thanks!

File: _includes/catalogpage.html (hunk 1 above)
Comment (Hixie) [inline _includes/catalogpage.html]: I changed the JSON file too.

**Comment by Hixie** [comment]
Isn't sending PRs from a forked repo the way one is supposed to do things?

File: _includes/catalogpage.html (hunk 1 above)
Comment (sethladd) [inline _includes/catalogpage.html]: Ah, thanks! That diff was collapsed.

**Comment by sethladd** [comment]
lgtm

**Pull request merged**